Makefile

```diff
@@ -63,7 +63,7 @@ uninstall:
 	rm /etc/systemd/system/flagd.service
 	rm -f $(DESTDIR)$(PREFIX)/bin/flagd
 lint:
-	go install -v github.com/golangci/golangci-lint/cmd/golangci-lint@latest
+	go install -v github.com/golangci/golangci-lint/cmd/golangci-lint@v1.55.2
```

Comment:
Before we forget, I created this to track - https://github.com/open-feature/flagd/issues/1207 

@thisthat @toddbaert  tasks for our todo list :)